packages/main/src/themes/sap_horizon/Card-parameters.css

```diff
@@ -1,10 +1,13 @@
 @import "../base/Card-parameters.css";
 
 :root {
-	--_ui5_card_hover_box_shadow: var(--sapContent_Shadow1);
+	--_ui5_card_hover_box_shadow: var(--sapContent_Shadow2);
 	--_ui5_card_header_focus_offset: 0px;
 	--_ui5_card_header_focus_radius: var(--_ui5_card_border-radius);
 	--_ui5_card_header_title_font_family: var(--sapFontHeaderFamily);
 	--_ui5_card_header_title_font_size: var(--sapFontHeader6Size);
-	--_ui5_card_header_hover_bg: var(--sapTile_Background);
-}
\ No newline at end of file
+	--_ui5_card_header_hover_bg: var(--sapTile_Hover_Background);
+	--_ui5_card_header_active_bg: var(--sapTile_Active_Background);
+	--_ui5_card_header_border: none;
+	--_ui5_card_border-radius: var(--sapTile_BorderCornerRadius);
```

Comment:
Seems like this variable `sapTile_BorderCornerRadius` is not defined. The rounded corners are gone.